src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -64,8 +64,8 @@
   "stsjorbsmod:DamnationPower": {
     "NAME": "Damnation",
     "DESCRIPTIONS": [
-      "Permanently #yTransform the next non-Legendary card played this combat.",
-      "Permanently #yTransform the next non-Legendary card played this combat, then #yUpgrade it."
+      "Permanently #yTransform the next non- [REMOVE_SPACE]#yLegendary card played this combat.",
+      "Permanently #yTransform the next non- [REMOVE_SPACE]#yLegendary card played this combat, then #yUpgrade it."
```

Comment:
by adding `non-legendary` as a valid keyword for `Legendary`, this can also probably change. I'm noticing that it was done for shrapnel bloom.